config/src/defaults.rs

```diff
@@ -204,6 +204,14 @@ pub trait Defaults {
     fn rocksdb_enable_statistics(&self) -> bool {
         false
     }
+
+    fn ntp_update_period(&self) -> i64 {
+        600
+    }
+
+    fn ntp_server(&self) -> Vec<String> {
+        vec!["0.pool.ntp.org:123".to_string()]
```

Comment:
It is recommended to use at least 4 NTP servers, let's add a few more.

Reply:
current library can not manage more than one library